sklearn/model_selection/_search.py

```diff
@@ -337,10 +340,12 @@ def _check_param_grid(param_grid):
                                  "list.")
 
 
+# XXX Remove in 0.20
 class _CVScoreTuple (namedtuple('_CVScoreTuple',
                                 ('parameters',
                                  'mean_validation_score',
-                                 'cv_validation_scores'))):
+                                 'cv_validation_scores',
+                                 'corrected_std'))):
```

Comment:
I feel we should just revert this back as it was. It was just available in the `__repr__` (unless of course if there is a simple solution)